### PR TITLE
[FIX] im_livechat: fix button translation for feedback

### PR DESCRIPTION
**Pull request by adrienmilis**
Before this commit, the "Send" button on the feedback form for livechat was not translatable.

This commit follows 92e95a1f4f8ea914b257e03200f537d8a4bb5e3c which fixed translations for those templates.

opw-3509309

### CODE REVIEW

**Comment by alexkuhn** [comment]
Hello, version 14.0 is no longer supported. Could you please make another PR that targets 15.0, which is the oldest currently supported version?

Also when changes in a stable version add/update a translatable string, we should manually put diff in the `POT` file of the module, so that translators are aware of this new term to (re-)translate.

Thanks!

**Comment by adrienmilis** [comment]
Hello @alexkuhn :slightly_smiling_face: 

This is part of a bugfix ticket that was opened in September when 14.0 was still supported, so the client's database is in 14.0. Do you think that in that case it could still be merged?

If so, I'll update the `POT` file.

Thanks!

**Comment by alexkuhn** [comment]
Ok for an exceptional fix in 14.0. However you still need to make another equivalent PR that targets at least 15.0 to be taken into account by fwbot. Thanks!

**Comment by adrienmilis** [comment]
@alexkuhn okay thanks - I've updated this PR with the translation and 15.0 PR is available at https://github.com/odoo/odoo/pull/145550

File: addons/im_livechat/static/src/legacy/public_livechat.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
             <div class="o_livechat_rating_reason">
                 <textarea id="reason" placeholder="Explain your note"></textarea>
                 <div class="o_livechat_rating_reason_button">
-                    <input type="button" class="btn btn-primary btn-sm o_rating_submit_button" value="Send" />
+                    <button type="button" class="btn btn-primary btn-sm o_rating_submit_button">Send</button>
```
Comment (alexkuhn) [inline addons/im_livechat/static/src/legacy/public_livechat.xml:20]: What's the reason to change `input` from `button`?
It doesn't seem to be related to the issue at hand that it's not translated properly. It looks more a style fix that is not documented in commit message/title. Do you confirm this is the reasoning of change?

Comment (alexkuhn) [reply addons/im_livechat/static/src/legacy/public_livechat.xml:20]: Oh right, makes sense then :)

**Review by alexkuhn — APPROVED** [review]
@robodoo delegate+ r+
